docs/tutorials/edc-cut-bell-pair-benchmarking.ipynb

```diff
@@ -0,0 +1,715 @@
+    "\n",
+    "In the [reference experiment](#references), the authors introduce a Bell pair fidelity benchmark to identify which parts of a device are best suited for LOCC-based entanglement. The idea is to run a small dynamic circuit on every group of four connected qubits in the processor. This four-qubit circuit first creates a Bell pair on two middle qubits, then uses those as a resource to entangle the two edge qubits via LOCC. Concretely, qubits 1 and 2 are prepared into an uncut Bell pair locally (using a Hadamard and CNOT), and then a teleportation routine consumes that Bell pair to entangle qubits 0 and 3. Qubits 1 and 2 are measured during execution of the circuit, and based on those outcomes, Pauli corrections (an X on qubit 3 and Z on qubit 0) are applied. Qubits 0 and 3 are then left in a Bell state at the end of the circuit.\n",
+    "\n",
+    "To quantify the quality of this final entangled pair, we measure its stabilizers: specifically the parity in the $Z$ basis ($Z_0Z_3$) and in the $X$ basis ($X_0X_3$). For a perfect Bell pair, both of these expectations equal +1. In practice, hardware noise will reduce these values. We therefore repeat the circuit twice for each qubit-pair: one circuit measures qubits 0 and 3 in the $Z$ basis, and another measures them in the $X$ basis. From the results, we obtain an estimate of $\\langle Z_0Z_3\\rangle$ and $\\langle X_0X_3\\rangle$ for that pair of qubits. We use the mean squared error (MSE) of these stabilizers with respect to the ideal value 1 as a simple metric of entanglement fidelity. A lower MSE means the two qubits achieved a Bell state closer to ideal (higher fidelity), whereas a higher MSE indicates more error. By scanning this experiment across the device, we can benchmark the measurement-and-feedforward capability of different qubit groups and identify the best pairs of qubits for LOCC operations.\n",
+    "\n",
+    "This tutorial demonstrates the experiment on an IBM Quantum device to illustrate how dynamic circuits can be used to generate and evaluate entanglement between distant qubits. We will map out all 4-qubit linear chains on the device, run the teleportation circuit on each, and then visualize the distribution of MSE values. This end-to-end procedure shows how to leverage Qiskit Runtime and dynamic circuit features to inform hardware-aware choices for cutting circuits or distributing quantum algorithms across a modular system."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "152c479f",
+   "metadata": {},
+   "source": [
+    "## Requirements\n",
+    "\n",
+    "Before starting this tutorial, ensure that you have the following installed:\n",
+    "\n",
+    "* Qiskit SDK 1.0 or later, with visualization support (`pip install 'qiskit[visualization]'`)\n",
```

Comment:
Is Qiskit 1.0 still OK, or do we want them using 2.0?

Reply:
Yeah we should probably bump all of the tutorials requirements to 2.0, which is what we tested them on at release